pkg/controller/compliance/compliance_controller.go

```diff
@@ -442,6 +442,9 @@ func (r *ReconcileCompliance) Reconcile(ctx context.Context, request reconcile.R
 	}
 
 	reqLogger.V(3).Info("rendering components")
+
+	namespaceComp := render.NewPassthrough(render.CreateNamespace(render.ComplianceNamespace, network.KubernetesProvider, render.PSSPrivileged))
```

Comment:
I'm not sure I understand the relevance of this change

Reply:
The certificates need to be created/updated before the compliance deployments are updated and to do that we need the namespace created first.